docs/rules/no-negated-condition.md

```diff
@@ -0,0 +1,65 @@
+# Disallow use of negated expressions in conditions (no-negated-condition)
+
+Checks against the use of a negated expression in an if condition when the else branch is empty or in a ternary operator. Negated conditions are more difficult to understand. Code can be made more readable by inverting the condition instead.
```

Comment:
Wait, I though this was fixed already. This rule checks if condition when the else branch is **not** empty.